sdk/communication/communication-chat/samples/typescript/src/messageOperations.ts

```diff
@@ -15,10 +18,9 @@ dotenv.config();
 
 export async function main() {
   const connectionString =
-    process.env["COMMUNICATION_CONNECTION_STRING"] ||
+    process.env["COMMUNICATION_SAMPLES_CONNECTION_STRING"] ||
     "endpoint=https://<resource-name>.communication.azure.com/;<access-key>";
-  const endpoint =
-    process.env["COMMUNICATION_ENDPOINT"] || "https://<resource-name>.communication.azure.com";
+  const endpoint = parseConnectionString(connectionString).endpoint;
```

Comment:
Why we need to change this?

Reply:
We have been using this pattern even for the other packages. In addition we don't have `COMMUNICATION_ENDPOINT` in the common sub config anymore. So this is actually a better approach